addons/l10n_it_edi_sale/tests/test_edi_sale_order_pa.py

```diff
@@ -0,0 +1,100 @@
+import datetime
+from freezegun import freeze_time
```

Comment:
Small detail, but:
```suggestion

from freezegun import freeze_time
```

https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 1. Related third party imports.
> 1. Local application/library specific imports.
>
> ***You should put a blank line between each group of imports.***